game/templates/game/board.html

```
@@ -293,8 +320,15 @@
             {% if user.is_authenticated %}
 
             <div class="profile-dropdown">
-                <button class="profile-btn">
-                    {{ user.username }} ⌄
+                <button class="profile-btn" style="display:flex;align-items:center;gap:4px;">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)